### PR TITLE
Rewrite installation selector JS

### PR DESCRIPTION
**Pull request by WardBrian**
This rewrites the install page such that:

1. The grid is generated by the JS. This, in particular, means we don't need to show so many disabled options all the time
2. The actual content is stored in the markdown file, which makes it easier to edit
3. I started writing the actual content

Screenshot:
image

### CODE REVIEW

File: install/install.qmd [hunk 1]
```diff
@@ -89,6 +89,12 @@ For more information, see the
...
 :::::
 
+::::: {.install #install-cmdstanr-conda .hidden}
+Run `conda install -c conda-forge r-cmdstanr`.
+
+Note: this will also install CmdStan and any system prerequesites.
+:::::
+
 ::::: {.install #install-cmdstanr-github-src .hidden}
 In R, run `remotes::install_github("stan-dev/cmdstanr")`.
```
Comment (mitzimorris) [inline install/install.qmd:99]: instead of using remotes and stan-dev github, instructions should be to r-universe.

Comment (WardBrian) [reply install/install.qmd:99]: R-universe is the option two above this one

**Review by mitzimorris — COMMENTED** [review]
someone recently pointed out that the mathematica stan wrapper still works - https://github.com/stan-dev/cmdstan/pull/1291

at this point, pystan and mathematica stan are unsupported, and there's a message on the stan.jl repo:  https://github.com/StanJulia/Stan.jl/blob/master/README.md

information about what is currently supported and which methods are available (most notably pathfinder) should be displayed somewhere.   a different grid?

**Review by mitzimorris — APPROVED** [review]
as far as rewriting the js selector, this is great!

comments re more info can go into another PR

**Comment by WardBrian** [comment]
> information about what is currently supported and which methods are available (most notably pathfinder) should be displayed somewhere. a different grid?

I think anything that is unsupported should be outside the quick-install grid and be under the "Other packages" section, personally.

I also think information about what the packages do/which one is right for you if you don't already know should be separate from the how-to-install-stan information. Above the grid we could say `Not sure which interface is right for you? See our [Comparison Section]...`

**Comment by mitzimorris** [comment]
> I think anything that is unsupported should be outside the quick-install grid and be under the "Other packages" section, personally.

this now includes PyStan.

**Pull request merged**